### PR TITLE
New header implementation for pwiz backend

### PR DESCRIPTION
**Pull request by jotsetung**
This pull request contains the new implementations for the `getScanHeaderInfo` and `getAllScanHeaderInfo` of the `pwiz` backend.
It fixes issues #106 in `mzR` and https://github.com/lgatto/MSnbase/issues/216 in `MSnbase`.

I added also a unit test to ensure that the header that is read by the new implementations matches the one which would be read using the `Ramp` backend.

### CODE REVIEW

**Comment by lgatto** [comment]
This looks good to me. @sneumann, any comments before merging?

File: R/methods-mzRpwiz.R [hunk 1]
```diff
@@ -54,15 +54,22 @@ setMethod("header", c("mzRpwiz", "missing"),
 
 setMethod("header", c("mzRpwiz", "numeric"),
           function(object, scans) {
-            if (length(scans) == 1) {
+              if (length(scans) == 1) {
+                  res <- object@backend$getScanHeaderInfo(scans)
+                  ## Convert data.frame to list to be conform with old code
+                  return(as.list(res))
               return(object@backend$getScanHeaderInfo(scans))
```
Comment (sneumann) [inline R/methods-mzRpwiz.R]: Is this line ever reached ? If not, should it go ?

Comment (lgatto) [inline R/methods-mzRpwiz.R]: Indeed!

Comment (jotsetung) [inline R/methods-mzRpwiz.R]: thanks @sneumann for spotting this! I fixed that and pushed the update.

**Comment by jotsetung** [comment]
Hi @lgatto and @sneumann , sorry, I don't want to  push stuff, but any chance to merge this pull request? 
This would solve the `MSnbase` check timeout on the build machines (https://github.com/lgatto/MSnbase/issues/216)

**Comment by lgatto** [comment]
I assume the failure is not related to R but travis. Will address that later. Merging now. Thanks @jotsetung

**Comment by jotsetung** [comment]
Thanks @lgatto . I ran the checks on my computer and did not get any errors - so I guess they are unrelated to the pull request.

**Comment by lgatto** [comment]
Also pushed to Bioc.

**Pull request merged**